ignite-base/App/Themes/ApplicationStyles.js

```diff
@@ -34,13 +34,15 @@ const ApplicationStyles = {
       color: Colors.snow,
       marginVertical: Metrics.smallMargin,
       textAlign: 'center',
-      fontFamily: Fonts.type.bold
+      fontFamily: Fonts.type.bold,
+      backgroundColor: Colors.transparent
     },
     subtitle: {
       color: Colors.snow,
       padding: 5,
       marginBottom: 5,
-      marginHorizontal: Metrics.smallMargin
+      marginHorizontal: Metrics.smallMargin,
+      backgroundColor: Colors.transparent
```

Comment:
why did you have to do this?   I'd like to get away from Text having default background colors that have to be overridden.